From 034835877f8e08af565b7f58462a5fad824c33e5 Mon Sep 17 00:00:00 2001
From: Astro <astro@spaceboyz.net>
Date: Fri, 19 Apr 2024 19:00:26 +0200
Subject: [PATCH 1/2] compat-cxx-safe-header

---
 tools/compat/nbtool_config.h.in | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/tools/compat/nbtool_config.h.in b/tools/compat/nbtool_config.h.in
index c1dd6c5a7f0d..f4f03eaaddc1 100644
--- a/tools/compat/nbtool_config.h.in
+++ b/tools/compat/nbtool_config.h.in
@@ -927,5 +927,14 @@
 /* Define if you have u_int8_t, but not uint8_t. */
 #undef uint8_t
 
+#ifdef __cplusplus
+extern "C" {
+#endif
+
 #include "compat_defs.h"
+
+#ifdef __cplusplus
+}
+#endif
+
 #endif /* !__NETBSD_NBTOOL_CONFIG_H__ */
-- 
2.42.0

