From 3b25c2b2a00722de88c6b8b26d8acf954e23849a Mon Sep 17 00:00:00 2001
From: John Ericson <John.Ericson@Obsidian.Systems>
Date: Fri, 19 Apr 2024 19:03:07 +0200
Subject: [PATCH 2/2] Make should not hit configure

---
 tools/compat/Makefile | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/tools/compat/Makefile b/tools/compat/Makefile
index 1149fb31c333..4e278c1d9d55 100644
--- a/tools/compat/Makefile
+++ b/tools/compat/Makefile
@@ -82,12 +82,6 @@ CONFIGURE_ARGS=	--cache-file=config.cache
 CONFIGURE_ARGS+=--silent
 .endif
 
-config.cache: include/.stamp configure nbtool_config.h.in defs.mk.in
-	rm -f ${.TARGET}
-	CC=${HOST_CC:Q} CFLAGS=${HOST_CFLAGS:Q} CPPFLAGS=${HOST_CPPFLAGS:Q} \
-		LDFLAGS=${HOST_LDFLAGS:Q} \
-		${HOST_SH} ${.CURDIR}/configure ${CONFIGURE_ARGS}
-
 defs.mk: config.cache
 	@touch ${.TARGET}
 
-- 
2.42.0

